index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <label>Enter number</label>
+    <input id="users-value" oninput="humanizeValue('users-value')">
```

Comment:
1. use addEventListener and tell me why it is better than inline on-handlers
2. use event object instead of hardcoded id
https://developer.mozilla.org/en-US/docs/Web/Events/Event_handlers#eventtarget.addeventlistener